### DISCUSSION TITLE
how do i unlock pack a punch on House?

### DISCUSSION THREAD

**Discussion by leoereno** [nzp-team/nzportable#903] [General]
maybe im just dumb but i can't find pap on House map

**Comment by SlimoIsHere** [comment]
You have to find and shoot 3 teddy bears around he map to unlock PaP (:

**Reply by leoereno** [reply]
i did that but nothing happened, is there an specific order i have to shoot them?

**Reply by DerpedCrusader** [reply]
Incorrect, it's available as soon as you get upstairs 

I'll give you a hint, the pap machine makes noise, try finding it that way ;)

**Reply by leoereno** [reply]
i'm shooting all the 3 teddy bears (1 on spawn, 1 on truck and 1 on the roof upstairs) but I swear all it does is turn on the map's song. I also can't hear the pap's ticking noise anywhere :/

**Reply by SlimoIsHere** [reply]
My bad, haven't played the map in a while lol

**Reply by PepsiNun** [reply]
There's a illusionary wall on the 3rd floor on the left of where you buy the browning,
you can just walk through it. 
The secret room also has a random buff drink dispenser.

You don't have to shoot the bears they're just for the music

(was wondering the same thing so i opened the map in a map editor)

Here's a gif of where to find it.


![secretroom](https://github.com/user-attachments/assets/28e59870-b25e-4437-a896-e40e6ffd359b)

**Reply by leoereno** [reply]
awesome thank you so much!
